src/comm_manager.cpp

```diff
@@ -249,6 +214,8 @@ void CommManager::command_callback(CommLink::Command command)
     case CommLink::Command::COMMAND_SEND_VERSION:
       comm_link_.send_version(sysid_, GIT_VERSION_STRING);
       break;
+    default:
```

Comment:
This is one instance where we actually don't want a default case, because we want the compiler to guarantee for us that all possible commands are handled explicitly

Reply:
Strict compilation requires default, perhaps we change that flag?